src/topology/continuous_function/algebra.lean

```diff
@@ -453,6 +453,25 @@ end
 
 end continuous_map
 
+lemma continuous_map.subsingleton_subalgebra_ext {α : Type*} [topological_space α]
+  {R : Type*} [comm_semiring R] [topological_space R] [topological_semiring R]
+  [subsingleton α] (s₁ s₂ : subalgebra R C(α, R)) :
+  s₁ = s₂ :=
```

Comment:
Shouldn't the statement be:
```suggestion
-- TODO[gh-6025]: make this an instance once safe to do so
lemma continuous_map.subsingleton_subalgebra {α : Type*} [topological_space α]
  {R : Type*} [comm_semiring R] [topological_space R] [topological_semiring R]
  [subsingleton α] : subsingleton (subalgebra R C(α, R)) :=
```

Reply:
Sure! I was thinking of it was an `ext` lemma, but this is better.